ajax/libs/tributejs/1.0.0/tribute.min.js.map

```diff
@@ -0,0 +1 @@
,QAC5CsN,EAASvB,GAAKwB,EACdD,EAASnN,YAAY9D,KAAK0D,cAAcgO,eAAeV,IACvDvN,EAAMmJ,WAAWqE,GACjBnG,EAAIM,kBACJN,EAAIQ,SAASmG,EAEb,IAAI3H,IACFO,KAAM,EACND,IAAK6G,EAASU,aAMhB,OAHA3R,MAAK6Q,yBAAyBI,EAAUnH,GAExCmH,EAASjE,WAAW8D,YAAYG,GACzBnH,KAGTrK,IAAK,iBACLQ,MAAO,SAAwByK,GAM7B,IALA,GAAIkH,GAAmB,GACnBC,EAAalR,OACbmR,EAAwB,IACxB/I,EAAI2B,EAEc/J,SAAfkR,GAAkD,IAAtBA,EAAWxB,QAG5C,GAFAwB,EAAa9I,EAAEgJ,wBAEW,IAAtBF,EAAWxB,SACbtH,EAAIA,EAAE4B,WAAW,GACPhK,SAANoI,IAAoBA,EAAEgJ,uBACxB,MAKN,IAAIC,GAAUH,EAAWzH,IACrB6H,EAAaD,EAAUH,EAAWxB,MAEtC,IAAI2B,EAAU,EACZhM,OAAOkM,SAAS,EAAGlM,OAAOmM,YAAcN,EAAWzH,IAAMwH,OACpD,IAAIK,EAAajM,OAAOoM,YAAa,CAC1C,GAAIC,GAAOrM,OAAOmM,YAAcN,EAAWzH,IAAMwH,CAE7CS,GAAOrM,OAAOmM,YAAcL,IAC9BO,EAAOrM,OAAOmM,YAAcL,EAG9B,IAAIQ,GAAUtM,OAAOmM,aAAenM,OAAOoM,YAAcH,EAErDK,GAAUD,IACZC,EAAUD,GAGZrM,OAAOkM,SAAS,EAAGI,QAKlBhQ,KAKLG,EAAgB,WAClB,QAASA,GAAckD,GACrBvH,gBAAgB4B,KAAMyC,GAEtBzC,KAAK2F,QAAUA,EACf3F,KAAK2F,QAAQpB,OAASvE,KA2JxB,MAxJAnB,cAAa4D,IACXhD,IAAK,eACLQ,MAAO,SAAsBsS,EAASC,GACpC,GAAIC,GAASzS,IAEb,OAAOwS,GAAMhO,OAAO,SAAUkB,GAC5B,MAAO+M,GAAOjE,KAAK+D,EAAS7M,QAIhCjG,IAAK,OACLQ,MAAO,SAAcsS,EAAS7M,GAC5B,MAAuC,QAAhC1F,KAAK0S,MAAMH,EAAS7M,MAG7BjG,IAAK,QACLQ,MAAO,SAAesS,EAAS7M,EAAQiN,GACrCA,EAAOA,KACP,IAKIlO,IAHMiB,EAAOxG,OAGPyT,EAAKlO,KAAO,IAClBC,EAAOiO,EAAKjO,MAAQ,GACpBkO,EAAgBD,EAAKE,eAAiBnN,GAAUA,EAAO8B,aAI3D+K,GAAUI,EAAKE,eAAiBN,GAAWA,EAAQ/K,aAEnD,IAAIsL,GAAe9S,KAAK+S,SAASH,EAAeL,EAAS,EAAG,KAC5D,OAAKO,IAKHE,SAAUhT,KAAKiT,OAAOvN,EAAQoN,EAAaI,MAAOzO,EAAKC,GACvDyO,MAAOL,EAAaK,OALb,QASX1T,IAAK,WACLQ,MAAO,SAAkByF,EAAQ6M,EAASa,EAAaC,EAAcP,GAEnE,GAAIP,EAAQrT,SAAWmU,EAGrB,OACEF,MAAOnT,KAAKsT,eAAeR,GAC3BI,MAAOJ,EAAaS,QAKxB,MAAI7N,EAAOxG,SAAWkU,GAAeb,EAAQrT,OAASmU,EAAe3N,EAAOxG,OAASkU,GAArF,CASA,IALA,GAAI/E,GAAIkE,EAAQc,GACZrO,EAAQU,EAAOpC,QAAQ+K,EAAG+E,GAC1BI,EAAO7S,OACP8S,EAAO9S,OAEJqE,GAAQ,GAAI,CAMjB,GALA8N,EAAa3F,KAAKnI,GAClByO,EAAOzT,KAAK+S,SAASrN,EAAQ6M,EAASvN,EAAQ,EAAGqO,EAAe,EAAGP,GACnEA,EAAaY,OAGRD,EACH,MAAOD,KAGJA,GAAQA,EAAKL,MAAQM,EAAKN,SAC7BK,EAAOC,GAGTzO,EAAQU,EAAOpC,QAAQ+K,EAAGrJ,EAAQ,GAGpC,MAAOwO,OAGT/T,IAAK,iBACLQ,MAAO,SAAwB6S,GAC7B,GAAIK,GAAQ,EACRM,EAAO,CAcX,OAZAX,GAAa/N,QAAQ,SAAUC,EAAO/F,GAChCA,EAAI,IACF6T,EAAa7T,EAAI,GAAK,IAAM+F,EAC9ByO,GAAQA,EAAO,EAEfA,EAAO,GAIXN,GAASM,IAGJN,KAGT1T,IAAK,SACLQ,MAAO,SAAgByF,EAAQiO,EAASlP,EAAKC,GAC3C,GAAIsO,GAAWtN,EAAOuG,UAAU,EAAG0H,EAAQ,GAM3C,OAJAA,GAAQ5O,QAAQ,SAAUC,EAAO/F,GAC/B+T,GAAYvO,EAAMiB,EAAOV,GAASN,EAAOgB,EAAOuG,UAAUjH,EAAQ,EAAG2O,EAAQ1U,EAAI,GAAK0U,EAAQ1U,EAAI,GAAKyG,EAAOxG,UAGzG8T,KAGTvT,IAAK,SACLQ,MAAO,SAAgBsS,EAASqB,EAAKjB,GACnC,GAAIkB,GAAU7T,IAGd,OADA2S,GAAOA,MACAiB,EAAIE,OAAO,SAAUC,EAAMpR,EAAS2L,EAAKsF,GAC9C,GAAII,GAAMrR,CAENgQ,GAAKhO,UACPqP,EAAMrB,EAAKhO,QAAQhC,GAEdqR,IAEHA,EAAM,IAIV,IAAIhB,GAAWa,EAAQnB,MAAMH,EAASyB,EAAKrB,EAW3C,OATgB,OAAZK,IACFe,EAAKA,EAAK7U,SACRwG,OAAQsN,EAASA,SACjBG,MAAOH,EAASG,MAChBnO,MAAOsJ,EACP9I,SAAU7C,IAIPoR,OACFE,KAAK,SAAUC,EAAGC,GACvB,GAAIC,GAAUD,EAAEhB,MAAQe,EAAEf,KAC1B,OAAIiB,GAAgBA,EACbF,EAAElP,MAAQmP,EAAEnP,YAKlBvC,IAGTuD,QAAO1F,QAAUA","file":"tribute.min.js"}
```

Comment:
@extend1994 I think old version don't need `.map` files. You can minify these files again using `minify.sh --no-map`